onnxruntime/core/framework/data_types.cc

```diff
@@ -619,10 +638,7 @@ void RegisterAllProtos(const std::function<void(MLDataType)>& reg_fn) {
   REGISTER_ONNX_PROTO(MapInt64ToFloat, reg_fn);
   REGISTER_ONNX_PROTO(MapInt64ToDouble, reg_fn);
 
-  REGISTER_ONNX_PROTO(VectorString, reg_fn);
-  REGISTER_ONNX_PROTO(VectorFloat, reg_fn);
-  REGISTER_ONNX_PROTO(VectorInt64, reg_fn);
-  REGISTER_ONNX_PROTO(VectorDouble, reg_fn);
+  REGISTER_SEQ_TENSOR_PROTO(float, reg_fn);
 
```

Comment:
What's the reason for removing these?

Reply:
Those were the original registrations that represented seq(tensor) as vector<float>, etc. However, this reminded me to register other types as well here. Will do.